cpu/driver/runtest.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Red Hat, Inc. All rights reserved.
+#
+# This copyrighted material is made available to anyone wishing
+# to use, modify, copy, or redistribute it subject to the terms
+# and conditions of the GNU General Public License version 2.
+#
+# This program is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied
+# warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+# PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public
+# License along with this program; if not, write to the Free
+# Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+# Boston, MA 02110-1301, USA.
+#
+
+FILE=$(readlink -f ${BASH_SOURCE})
+NAME=$(basename $FILE)
+CDIR=$(dirname $FILE)
+TEST=${TEST:-"$0"}
+TMPDIR=/var/tmp/$(date +"%Y%m%d%H%M%S")
+
+source ${CDIR%/cpu/driver}/cpu/common/libbkrm.sh
+source ${CDIR%/cpu/driver}/cpu/common/libutil.sh
+
+function verify_intel_cpufreq_driver
+{
+    rlLog "Start to verify intel cpu freq driver"
+
+    lscpu | grep " hwp " &> /dev/null
```

Comment:
hi @idorax I believe David said the spaces can be removed ?

Reply:
OK, the spaces are removed.